eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServer.java

```diff
@@ -131,6 +131,75 @@ public QueryResponse getVariantById(@PathVariable("variantId") String variantId,
         return setQueryResponse(queryResult);
     }
 
+    @RequestMapping(value = "/{variantIds}/infoByList", method = RequestMethod.GET)
```

Comment:
Sorry for the misunderstanding, here I meant that value would be `/{variantIds}` instead of `/{variantIds}/infoByList`